src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -365,8 +361,11 @@
                          (when (empty? receiver-network-values) :receiver-network-values)))})))
 
 (rf/reg-event-fx :wallet/get-suggested-routes
```

Comment:
Another thing is that this event was initially thought to be implemented for Send and Bridge, and as sending might involve bridging, it initially made sense to have it in a send-specific namespace. Not that we though a lot about this, but we initially implemented it here and never reconsider the decision.

But now that we know we need this to Swap transactions, I think it makes sense to move it to a common namespace for wallet where can be reused, and think of a more generic approach. If in the future we begin to support more types of transactions like ENSRegister or StickersBuy, this may start to be bigger a problem.

Reply:
I would be happy to start refactoring it but probably already after this release) Looks like a field for all kinds of errors)

Reply:
Yes let's hold off refactoring this for now. After the 2.30 release we should revisit it for sure!